ironic/resource_ironic_deployment.go

```diff
@@ -85,8 +93,42 @@ func resourceDeploymentCreate(d *schema.ResourceData, meta interface{}) error {
 
 	d.SetId(d.Get("node_uuid").(string))
 
+	userData := d.Get("user_data")
+	var ignConf map[string]interface{}
+	if err = json.Unmarshal([]byte(userData.(string)), &ignConf); err != nil {
+		panic(err)
+	}
+
+	// Fetch the full rendered ignition config
+	url := ignConf["ignition"].(map[string]interface{})["config"].(map[string]interface{})["append"].([]interface{})[0].(map[string]interface{})["source"].(string)
```

Comment:
In #37 we discussed this, and agreed that the URL and cert should be extracted in the installer, not here?

Reply:
Hi Steve, This is an old commit and it has been removed. Sorry for the confusion. I'll squash the changes for clarity.